src/braket/aws/aws_quantum_task.py

```diff
@@ -257,7 +257,10 @@ def state(self, use_cached_value: bool = False) -> str:
     def _status(self, use_cached_value=False):
         status = self.metadata(use_cached_value).get("status")
         if not use_cached_value and status in self.NO_RESULT_TERMINAL_STATES:
-            self._logger.warning(f"Task is in terminal state {status} and no result is available")
+            self._logger.warning(f"Task is in terminal state {status} and no result is available.")
+            if status == "FAILED":
+                failure_reason = self.metadata(use_cached_value).get("failureReason", "unknown.")
+                self._logger.warning(f"Task failure reason is {failure_reason}.")
```

Comment:
Do you have examples of real failure reasons? Will they make sense being plugged into a sentence organically like this, or should they be explicitly quoted to a customer?